src/minesweeper/use-cases/exposeCell/exposeCell.ts

```diff
@@ -0,0 +1,53 @@
+import { IRepositoryUseCase } from '@minesweeper/domain/data.repository';
+import { IPosition, IBoardId, PublicBoardModel, Cell } from '@minesweeper/domain/models';
+import { getAdjacentVertex, hasSamePosition } from '@minesweeper/utils';
+import { createNeighborsCounter } from '../startGame/createNeighborsCounter/createNeighborsCounter';
+import { runExposeCellValidations, runExposeCellCases } from './exposeCell.validations';
+
+export interface IExposeCellProps extends IPosition, IRepositoryUseCase, IBoardId {}
+export type IExposeCellUseCase = (props: IExposeCellProps) => Promise<PublicBoardModel>;
+
+/**
+ * [Use Case] Mark flag in a cell in the board in a specific
+ * position to signal that there should be a bomb there.
+ */
+export const exposeCellUseCase: IExposeCellUseCase = async props => {
+  const { boardId, dataRepository, ...positionProps } = props;
+  const board = await dataRepository.getBoard({ boardId });
+
+  runExposeCellValidations(props, board);
+
+  const neighborsCounter = createNeighborsCounter({
```

Comment:
Here you can use the board property of "neighBorsBombsCounter" instead of function "createNeighborsCounter"

```suggestion
  const neighborsCounter = board.neighBorsBombsCounter;
```